ffi/capi/dart/package/lib/src/lib.g.dart

```diff
@@ -9720,12 +9720,10 @@ class ScriptWithExtensionsBorrowed implements ffi.Finalizable {
   // ignore: non_constant_identifier_names
   static final _ICU4XScriptWithExtensionsBorrowed_get_script_extensions_val =
       _capi<
-                  ffi
```

Comment:
Why is there a diff in a .dart file?

Reply:
dart regen runs dart fmt

Reply:
Looks like CI pins a different dart compiler. Undone.